x/stakeibc/keeper/transfer.go

```diff
@@ -147,43 +98,20 @@ func (k Keeper) TransferHoldingToCommunityPoolReturn(ctx sdk.Context, hostZone t
 		coin, msgTransferResponse)
 	k.Logger(ctx).Info(result)
 
-	// If there was no error in sending the transfer msg, store what the transferred denom will be in callback with amount
-	callbackArgs := types.CommunityPoolReturnTransferCallback{
-		HostZoneId:  hostZone.ChainId,
-		DenomStride: coin.Denom,
-		IbcDenom:    k.GetDenomOnHostZone(coin.Denom, hostZone),
-		Amount:      coin.Amount,
-	}
-	callbackArgsBz, err := proto.Marshal(&callbackArgs)
-	if err != nil {
-		return errorsmod.Wrapf(err, "Unable to marshal pool return transfer callback %+v", callbackArgs)
-	}
-
-	// Register a callback by hand when the transfer msg gets the completed ack, different callback for each coin in module
-	k.ICACallbacksKeeper.SetCallbackData(ctx, icacallbackstypes.CallbackData{
-		CallbackKey:  icacallbackstypes.PacketID(msg.SourcePort, msg.SourceChannel, msgTransferResponse.Sequence),
-		PortId:       msg.SourcePort,
-		ChannelId:    msg.SourceChannel,
-		Sequence:     msgTransferResponse.Sequence,
-		CallbackId:   ICACallbackID_CommunityPoolReturn,
-		CallbackArgs: callbackArgsBz,
-	})
-
 	return nil
 }
 
-// helper function to find the ibc denom on the foreign chain of tokens after transfer from stride
-func (k Keeper) GetDenomOnHostZone(strideDenom string, hostZone types.HostZone) (ibcDenom string) {
-	// we use the hostZone.TransferChannelId because direction is stride to hostZone
-	sourcePrefix := transfertypes.GetDenomPrefix(transfertypes.PortID, hostZone.TransferChannelId)
-	prefixedDenom := sourcePrefix + strideDenom
-
-	return transfertypes.ParseDenomTrace(prefixedDenom).IBCDenom()
-}
-
 // given a hostZone with native denom, returns the ibc denom on the zone for the staked stDenom
-func (k Keeper) GetStakedHostTokenDenomOnHostZone(hostZone types.HostZone) (ibcStakedDenom string) {
+func (k Keeper) GetStakedDenomOnHostZone(ctx sdk.Context, hostZone types.HostZone) (ibcStakedDenom string) {
```

Comment:
similarly, would recommend something like: `GetStIBCDenomOnHostZone` since "staked" sounds more like the native token